src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -550,23 +550,32 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
             End If
 
             Dim Here = 1
-            While CanGet(Here)
-                ch = Peek(Here)
-                If IsWhitespace(ch) Then
-                    Here += 1
-                Else
-                    Exit While
+            Dim atNewLine As Boolean
+            If CanGet(Here + 1) AndAlso IsWhitespace(Peek(Here)) AndAlso IsSingleQuote(Peek(Here + 1)) Then
+                tList.Add(MakeLineContinuationTrivia(GetText(1)))
```

Comment:
The syntax produced here by `MakeLineContinuationTrivia` should be run through `CheckFeatureAvailability` so that it's tagged as an error when the parser encounters it but the LangVersion is sufficiently old.